vsc_pdf_template_editor/example/assets/test/partions_inline_span_tpl_test.json

```diff
@@ -0,0 +1,225 @@
+{
+  "className": "TplDocument",
+  "pageMode": "'none'",
+  "children": [
+    {
+      "className": "TplPage",
+      "pageFormat": {
+        "format": "'letter'"
+      },
+      "child": {
+        "className": "TplPartitions",
```

Comment:
Partition and Partitions have to do with groups of widgets which can be broken across multiple pages. They work within a MultiPage widget. See https://pub.dev/documentation/pdf/latest/widgets/MultiPage-class.html. We need a separate test for partitions demonstrating that they can span two pages.